bitops.config.yaml

```
@@ -13,26 +13,7 @@ bitops:
     filename: bitops-run      # log filename
     err: bitops.logs          # error logs filename
     path: /var/logs/bitops    # path to log folder
-  opsrepo_root_default_dir: _default
-  plugins:    
-    aws:
-      source: https://github.com/bitops-plugins/aws
-    terraform:
-      source: https://github.com/bitops-plugins/terraform
-    cloudformation:
-      source: https://github.com/bitops-plugins/cloudformation
-    helm:
-      source: https://github.com/bitops-plugins/helm
-    kubectl:
-      source: https://github.com/bitops-plugins/kubectl
-    ansible:
-      source: https://github.com/bitops-plugins/ansible
-  deployments:
-    cloudformation:
-      plugin: cloudformation
-    terraform:
-      plugin: terraform
-    helm:
-      plugin: helm
-    ansible:
-      plugin: ansible
+  
+  default_folder: _default
```

Looking at other variables `bitops_envroot_dir`, `bitops_operations_dir`, `bitops_scripts_dir` I suggest to follow that consistent naming (`dir`, instead of `folder`):
```suggestion
  default_dir: _default
```

---

👍 I agree